test/apiv2/python/rest_api/test_v2_0_0_container.py

```diff
@@ -25,6 +32,22 @@ def test_inspect(self):
         self.assertId(r.content)
         _ = parse(r.json()["Created"])
 
+        r = requests.post(
+            self.podman_url + "/v1.40/containers/create?name=topcontainer",
+            json={"Cmd": ["top"], "Image": "alpine:latest"},
+        )
+        self.assertEqual(r.status_code, 201, r.text)
+        payload = r.json()
+        container_id = payload["Id"]
+        self.assertIsNotNone(container_id)
+
+        r = requests.get(self.podman_url + f"/v1.40/containers/{payload['Id']}/json")
```

Comment:
would it be worthwhile changing "v1.40" to a variable?  Seems like you have it in at least a few places.

Reply:
@TomSweeneyRedHat seems like a good idea to me, should probably run it by @jwhonce first. Would probably have to edit api_testcase.py and just add it to the end of the podman_url variable in there.